docs/sphinx/source/conf.py

```diff
@@ -80,6 +80,7 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static', '_images']
+smartquotes = False
```

Comment:
For future reference, this is because list entries starting with a quoted string don't render the open-quote character correctly when smartquotes is enabled, see https://github.com/sphinx-doc/sphinx/issues/4028